### ISSUE TITLE
PR template 

### ISSUE THREAD

**Issue by trallard** [jupyter/repo2docker#671]
Hey 👋🏼 

### Proposed change

With the recent merge of #654 and #655  I think we should also include templates for PR

Why?
Having standard templates will not only make it easier for those reviewing them to track the state of the PR but also will encourage the submitters to make sure they adhere to the contributing guidelines and make some checks before the PR is sent 

### Alternative options

--

### Who would use this feature?

Contributors to the project

### Who can do this work?

Low effort -  anyone could take this as this is a non-coding activity.
I can even take this as I already have some templates that I have for personal use/projects

**Comment by betatim** [comment]
Sounds like a good idea. Do you want to propose one? My only experience/recommendation is that we try and keep it short and not include things in a checklist that CI can/does check for us, let's bundle those as "make sure the checks all pass" if we feel that would help people. On a previous project we had a (at the end when we decided to remove it) checklist of seven or so items and most of the time people ignored the whole list :-/

**Comment by manics** [comment]
I opened a similar issue on team-compass for an org-wide template https://github.com/jupyterhub/team-compass/issues/302
(and despite intending to work on it straight away I've failed to do so....)

**Comment by trallard** [comment]
This also slipped my to-do list. Want to work on this together @manics ?

**Comment by manics** [comment]
Definitely! How about if I open a PR on https://github.com/jupyterhub/.github/ based on the comments on https://github.com/jupyterhub/team-compass/issues/302 and we go from there?
